app/serializers/move_serializer.rb

```diff
@@ -19,15 +23,17 @@ class MoveSerializer < ActiveModel::Serializer
              :date_from,
              :date_to
 
-  has_one :person, serializer: PersonSerializer
-  has_one :profile, serializer: ProfileSerializer # <- TODO: update the serializer
+  has_one :person
+  has_one :profile # <- TODO: update the serializer to V2
```

Comment:
This comment is actually redundant now as we ended up creating a V2 move action which just uses the v2 move serializer. We'll just remove this when the suppliers are on v2

Reply:
Fair enough, I'll remove the comment.